README.md

```diff
@@ -54,18 +55,34 @@ If you want to adjust the cron expression, please remember to schedule your jobs
 
 ## Action Spec
 
+### `leaderboardId`
+
+**Optional**
+
+Your Advent of Code leaderboard ID.
+To get this, go to your Go to [leaderboard](https://adventofcode.com/2020/leaderboard/private) and press 'View'.
+The leaderboard ID is at the end of the URL:
+
+```
+https://adventofcode.com/2021/leaderboard/private/view/(leaderboard ID)
+```
+
+If not provided it will be derived from the userId parameter
+
```

Comment:
Nit: could this be moved after the required fields, and adopt consistent text formatting with the other optional fields?

e.g.

```markdown
### `tableMarker`

*Optional* - default `<!--- advent_readme_stars table --->`

Description...
```